### PR TITLE
Component Accessibility Status: EAC Default state baseline audit/coverage

### LINKED ISSUE #13746: Component Accessibility Status: EAC Default state baseline audit/coverage
### Acceptance criteria

- [x] There are no violations when running the automated [Equal Access Checker](https://www.ibm.com/able/toolkit/tools) on default component state.
- [x] Make sure we have the default state baselines for all the components, especially any new components or enhancement to existing components since Josh [first implemented the EAC](https://github.com/carbon-design-system/carbon/issues/9475).

```[tasklist]
### Components missing avt in e2e files
- [x] ClassPrefix
- [x] ErrorBoundary
- [x] FluidForm
- [x] Grid
- [x] Heading
- [x] IconButton
- [x] IdPrefix
- [x] OverFlowMenuV2
```

These two components are waiting changes on this [issue](https://github.com/IBMa/equal-access/issues/761)
- [ ] Slider
- [ ] Toggle

### PR DESCRIPTION
**Pull request by guidari**
Closes #13746

We checked for the `avt` and `vrt` tests within the "e2e/components" folder and included all tests that were missing.

Also the cypress library was removed. So all files and imports related to that library were deleted.

#### Testing / Reviewing

- Run the command `yarn playwright test` in the root and hopefully all the tests will pass.

- Run `yarn build` to make sure the build still pass without the cypress library.

### CODE REVIEW

**Review by alisonjoseph — APPROVED** [review]
🥳

File: packages/react/src/components/OverflowMenuV2/index.js [hunk 1]
```diff
@@ -64,6 +65,10 @@ function OverflowMenuV2({
 }
 
 OverflowMenuV2.propTypes = {
+  /**
+   * Specify element's id to be used as the label for this OverflowMenu
+   */
+  'aria-labelledby': PropTypes.string,
```
Comment (francinelucca) [inline packages/react/src/components/OverflowMenuV2/index.js]: is this necessary here? I don't see it being used in the component 😅

**Review by francinelucca — CHANGES_REQUESTED** [review]
Mostly LGTM, just one question on the changes in OverflowMenuV2. Also looks like there's some conflicts after Jan's bring menu to stable PR merged in @guidari @andreancardona.

FYI I see 3 avt failures locally but they're not reflected upstream 🤔
image

**Review by francinelucca — APPROVED** [review]
Confirmed locally with @guidari it runs successfully on his machine. Mine runs considerably slower and I think some tests are timing out due to that. This should be good to merge as I see no errors upstream

**Pull request merged**